### DISCUSSION TITLE
Fade in & Fade out during playing single

### DISCUSSION THREAD

**Discussion by wegGehamstert** [savonet/liquidsoap#2835] [Q&A]
Hey, I wrote myself a script that plays a specific jingle at a specific time. This one plays over the other song.
I'd like to either play the jingle alone at the designated time, or massively turn down the volume on the song

This is my config for now:

```
clock   = single("~/clock/00_uhr.mp3")
radio = add([radio, switch([({0h-14h16},clock)])])
```

**Comment by toots** [comment]
You should have a look at `smooth_add`: https://www.liquidsoap.info/doc-2.1.3/reference-extras.html#smooth_add

**Reply by wegGehamstert** [reply]
I'm using Azuracast with Liquidsoap, so i dont know how to use this sadly. Is it just

```
(?duration : float, ?p : {float},
 normal : source(audio=pcm('a), video=internal('b),
 midi=internal('c)), special : source(audio=pcm('a),
 video=internal('b), midi=internal('c))) ->
source(audio=pcm('a), video=internal('b),
midi=internal('c))
```

wich i need to add to the bottom and change it?

I would love to have a quick tutorial, because im very new to Liquidsoap and dont understand everything

**Reply by wegGehamstert** [reply]
I want to bump this question, do you may help me how to do this in the custom liquidsoap config in azuracast?
